File: SMF1/src/SlimBundle/Entity/Vote.php
<?php

namespace SlimBundle\Entity;

/**
 * Vote
 */
class Vote
{
    /**
     * @var integer
     */
    private $note;

    /**
     * @var string
     */
    private $nomproduit;
    /**
     * @var string
     */
    private $commentaire;

    /**
     * @var integer
     */
    private $idvote;

    /**
     * @var integer
     */
    private $idutilisateur;


    /**
     * Set note
     *
     * @param integer $note
     *
     * @return Vote
     */
    public function setNote($note)
    {
        $this->note = $note;

        return $this;
    }

    /**
     * Get note
     *
     * @return integer
     */
    public function getNote()
    {
        return $this->note;
    }


    /**
     * Get idvote
     *
     * @return integer
     */
    public function getIdvote()
    {
        return $this->idvote;
    }

    /**
     * Set idutilisateur
     *
     * @param  $idutilisateur
     *
     * @return Vote
     */
    public function setIdutilisateur( $idutilisateur)
    {
        $this->idutilisateur = $idutilisateur;

        return $this;
    }

    /**
     * @return integer
     */
    public function getIdutilisateur()
    {
        return $this->idutilisateur;
    }

    /**
     * @return string
     */
    public function getNomproduit()
    {
        return $this->nomproduit;
    }

    /**
     * @param string $nomproduit
     */
    public function setNomproduit($nomproduit)
    {
        $this->nomproduit = $nomproduit;
    }

    /**
     * @return string
     */
    public function getCommentaire()
    {
        return $this->commentaire;
    }

    /**
     * @param string $commentaire
     */
    public function setCommentaire($commentaire)
    {
        $this->commentaire = $commentaire;
    }
}

File: ReclamationMobile/src/com/mycompany/Entite/Vote.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package com.mycompany.Entite;

/**
 *
 * @author pc hp
 */
public class Vote {
    public int idvote;
    public int idutilisateur;
    public String nomproduit;
    public int note;
    public String commentaire;

    public Vote(int idvote, int idutilisateur, String nomproduit, int note, String commentaire) {
        this.idvote = idvote;
        this.idutilisateur = idutilisateur;
        this.nomproduit = nomproduit;
        this.note = note;
        this.commentaire = commentaire;
    }

    public Vote(String nomproduit, int note, String commentaire) {
        this.nomproduit = nomproduit;
        this.note = note;
        this.commentaire = commentaire;
    }
 String Newline=System.getProperty("line.separator");
    @Override
    public String toString() {
        return "NomProduit = "+nomproduit+Newline+
                " Note = "+note+Newline+
                " Commentaire= "+commentaire+Newline;
    }
    public int getIdvote() {
        return idvote;
    }

    public void setIdvote(int idvote) {
        this.idvote = idvote;
    }

    public int getIdutilisateur() {
        return idutilisateur;
    }

    public void setIdutilisateur(int idutilisateur) {
        this.idutilisateur = idutilisateur;
    }

    public String getNomproduit() {
        return nomproduit;
    }

    public void setNomproduit(String nomproduit) {
        this.nomproduit = nomproduit;
    }

    public int getNote() {
        return note;
    }

    public void setNote(int note) {
        this.note = note;
    }

    public Vote() {
    }

    public Vote(int idvote, String nomproduit, int note, String commentaire) {
        this.idvote = idvote;
        this.nomproduit = nomproduit;
        this.note = note;
        this.commentaire = commentaire;
    }

    public String getCommentaire() {
        return commentaire;
    }

    public void setCommentaire(String commentaire) {
        this.commentaire = commentaire;
    }
    
    
    
    
}
